2024-01-18 08:56:13.149 [info] (Client) Pylance async client (2023.12.1) started with python extension (2023.22.1)
2024-01-18 08:56:14.146 [info] [Info  - 8:56:14 AM] (14562) Pylance language server 2023.12.1 (pyright a78e2b10) starting
2024-01-18 08:56:14.146 [info] [Info  - 8:56:14 AM] (14562) Server root directory: /home/devdragon/.vscode/extensions/ms-python.vscode-pylance-2023.12.1/dist
2024-01-18 08:56:14.228 [info] [Info  - 8:56:14 AM] (14562) Starting service instance "qtile"
2024-01-18 08:56:14.296 [info] [Info  - 8:56:14 AM] (14562) Setting pythonPath for service "qtile": "/bin/python"
2024-01-18 08:56:14.296 [info] [Info  - 8:56:14 AM] (14562) Setting environmentName for service "qtile": "3.11.6 (global)"
2024-01-18 08:56:14.333 [info] [Info  - 8:56:14 AM] (14562) Assuming Python version 3.11
2024-01-18 08:56:20.482 [info] [Info  - 8:56:20 AM] (14562) Found 10 source files
2024-01-18 08:56:24.513 [info] [Info  - 8:56:24 AM] (14562) [BG(1)] Long operation: checking: /home/devdragon/.config/qtile/settings/keys.py (2337ms)
2024-01-18 08:56:24.514 [info] [Info  - 8:56:24 AM] (14562) [BG(1)] Long operation: analyzing: /home/devdragon/.config/qtile/settings/keys.py (2550ms)
2024-01-18 08:56:45.064 [info] [Info  - 8:56:45 AM] (14562) [IDX(2)] Long operation: scan packages /home/devdragon/.config/qtile (23609ms)
2024-01-18 09:08:30.381 [info] [Error - 9:08:30 AM] Server process exited with code 0.
